internal/kafka/internal/migrations/20221109150000_rename_kafka_billing_model_column.go

```diff
@@ -0,0 +1,32 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+func renameKafkaBillingModelColumn() *gormigrate.Migration {
```

Comment:
This migration could result in intermittent `500` internal server errors if an incoming requests is served by an old pod. 
This is because the old pod will still be using the old column which won't exist by the time this migration has been executed: This is because since we do an equivalent of `select *` in most of the code bases. 

We could do a two phases rollout or just simply be aware of the error when this is being deployed and an incoming requests to list `/api/kafkas/v1/kafkas..` is made.

Reply:
We are aware of this but we decided to go ahead. The potential cases we can think is what you said about creating or listing kafka instances